spoc/spoc_dataset_builder.py

```diff
@@ -211,17 +232,12 @@ def _info(self) -> tfds.core.DatasetInfo:
                             dtype=np.float32,
                             doc="Indicates agent's location in the world coordinate frame. [XYZ, 3xeuler in degree]",
                         ),
-                        # TODO(kiana): how should BB info be stored?
-                        # 'manip_accurate_object_bbox': tfds.features.Scalar(
-                        #     dtype=np.bool_,
-                        #     doc='Indicates the bounding box for the target object in the manipulation camera, '
-                        #         'based on one method of calculation in simulation.',
-                        # ),
-                        # 'manip_task_relevant_object_bbox': tfds.features.Scalar(
-                        #     dtype=np.bool_,
-                        #     doc='Indicates the bounding box for the target object in the manipulation camera, '
-                        #         'based on a second method of calculation in simulation.',
-                        # ),
+                        'manip_object_bbox': tfds.features.Tensor(
+                            shape=(5,),
+                            dtype=np.float32,
+                            doc='Indicates the bounding box for the target object in the manipulation camera. '
+                                'values are [x1, y1, x2, y2, area]. [1000, 1000, 1000, 1000, 0] for no box.',
```

Comment:
are x1 etc in [0...1] or pixel values?

Reply:
Yes. Have clarified this now, thank you.